### PR TITLE
config settings to generate html cover coverage reports locally

### PR DESCRIPTION
**Pull request by amycheng**
some karma config changes so that HTML code coverage reports are generated when we run `npm run test-local`.

HTML reports would provide more immediate feedback than using coveralls.io

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -22,6 +22,9 @@ settings = {
         presets: ['es2015']
       }),
       istanbul({
+        instrumenterConfig: {
```
Comment (amycheng) [inline karma.conf.js]: fixes this [bug](https://github.com/karma-runner/karma-coverage/issues/157)

**Comment by byronhulcher** [comment]
:+1:

File: karma.conf.js (hunk 1 above)
Comment (byronhulcher) [inline karma.conf.js]: Would you mind adding a comment linking to this ?

**Comment by nelsonpecora** [comment]
Showing the coverage increases the coverage. There's something deeply philosophical about that. 👍

**Pull request merged**